css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - Breakpoints</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .card-wrapper {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+    @media (min-width: 576px) and (max-width:767px) {
```

Comment:
This is totally fine but you actually don't need `and (max-width:767px)` in your rule

Since the following rule will come into affect as soon as the width changes to 768px or higher

Reply:
oh yeah, that's right. Thanks!